tests/examples/opinions/united_states/bap9_example.compare.json

```diff
@@ -26,7 +26,7 @@
     "precedential_statuses": "Published",
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
-    "docket_numbers": "CC-19-1039-STaF CC-19-1040-STaF",
+    "docket_numbers": "CC-19-1039-STaF; CC-19-1040-STaF",
```

Comment:
for some courts where we collect oral arguments, modifying the `docket_number`  may cause the creation of a new docket, instead of updating the old one, since the lookup is done using docket number. On the other hand, multi docket number `docket_number`s are not common across sources, so maybe it will a limited problem

Reply:
I think multi docket numbers are very common across courts.